### PR TITLE
Database lesson1 home work  please review

### PR DESCRIPTION
**Pull request by RemyaManoharan**

### CODE REVIEW

File: databases/week1/lesson1.sql [hunk 1]
```diff
@@ -0,0 +1,38 @@
+SELECT count(id) FROM status;
```
Comment (veronicaburduzhan) [inline databases/week1/lesson1.sql]: COUNT command also should be written in uppercase

File: databases/week1/lesson1.sql [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+SELECT *
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+SELECT *
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name != 'Done';
```
Comment (veronicaburduzhan) [inline databases/week1/lesson1.sql]: In SQL we use the keyword NOT when result shouldn't be equal to something. Right syntax is this: 
```
SELECT * FROM <table>
WHERE NOT <column>=<"value">;
```

**Review by veronicaburduzhan — APPROVED** [review]
Great work 👍  You've complete all queries! I left two comments only

**Comment by RemyaManoharan** [comment]
Made changes based on the feedback. Thanks for the review :)

**Pull request merged**